app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,15 @@
+document.addEventListener(
+    'DOMContentLoaded',
+    function () {
+        let all_days = document.getElementsByClassName('day'), i;
+        for (i = 0; i < all_days.length; ++i) {
+            all_days[i].onclick = function () {
+                let daily_event = document.getElementById("day-view");
+                daily_event.classList.toggle("day-view-visible");
+            };
+        }
+    });
+
```

Comment:
Remove the blank lines

Reply:
Im doing a black line between function like we do at python, in JS all the code is connected ?